index.test.js

```diff
@@ -216,6 +216,20 @@ describe('validate-commit-msg.js', function() {
       expect(errors).to.deep.equal(['INVALID COMMIT MSG: subject does not match subject pattern!']);
       expect(logs).to.deep.equal([msg]);
     });
+
+    it('should lowercase type when autoFix is true and make it valid', function() {
+      m.config.autoFix = true;
```

Comment:
Sure, here's me talking about it for 3 minutes: https://www.briefs.fm/3-minutes-with-kent/27

These days I prefer using Jest with no `describe`, and using `test` rather than `it`. It's pretty amazing.